src/auxinfo/participant.rs

```diff
@@ -42,10 +41,9 @@ pub(crate) struct AuxInfoParticipant {
 }
 
 impl ProtocolParticipant for AuxInfoParticipant {
-    // Currently, AuxInfo puts output directly into persistent storage instead of
-    // returning it to the calling Participant.
-    // TODO #193: Update this type to actually define the output.
-    type Output = ();
+    // The output type includes `AuxInfoPublic` material for all participants
+    // (including ourselves) and `AuxInfoPrivate` for ourselves.
+    type Output = (Vec<AuxInfoPublic>, AuxInfoPrivate);
```

Comment:
question: Should we attach a session ID to these `Output`s, or is that the responsibility of the calling application? There's misuse potential if the caller doesn't carefully bind outputs to the session from whence they came, and ends up reusing them in incorrect contexts, but I'm not sure how well we can handle that from within.

Reply:
I'm going to stick with leaving it up to the calling application for now. Once we make progress on the public API, we can always add the session ID later.